### DISCUSSION TITLE
I would like to ask about this point: "Zero-Copy transfer support for non-plain types may be implemented in future releases of Fast DDS." Is there any relevant plan now?

### DISCUSSION THREAD

**Discussion by feiyouliang** [eProsima/Fast-DDS#3958] [Ideas]
I would like to ask about this point: "Zero-Copy transfer support for non-plain types may be implemented in future releases of Fast DDS." Is there any relevant plan now?

**Comment by JLBuenoLopez** [comment]
@feiyouliang 

Right now this feature is not in [Fast DDS current roadmap](https://github.com/eProsima/Fast-DDS/blob/master/roadmap.md). You might consider contacting [eProsima's commercial support team](https://github.com/eProsima/Fast-DDS#getting-help) for options to accelerate the development of this enhancement.
